order-karma.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Karma</title>
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/orderpage.css">
+    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <script>
+        function toggleMenu() {
+            var navBar=document.getElementById("navigation");
+            if (navBar.classList.contains("nav-mobile")) {
+                navBar.classList.remove("nav-mobile");
+                navBar.classList.add("navigation");
+            } else {
+                navBar.classList.remove("navigation");
+                navBar.classList.add("nav-mobile");
+            }
+            window.addEventListener("resize", function(event) {
+            if (navBar.classList.contains("nav-mobile")) {
+                navBar.classList.remove("nav-mobile");
+                navBar.classList.add("navigation");
+            }
+        })
+        }
+        
+    </script>
+</head>
+<body>
+
+    <header> 
+        <div class="logo">
+            <a href="./index.html">
+                <img src="./img/karma-logo.svg" alt="Karma Logo" />
+            </a>
+        </div>        
+        <nav>
+            <ul class="navigation" id="navigation">
+                <li><a href="./index.html" class="strong">Meet Karma</a></li>
+                <li><a href="#">How it Works</a></li>
+                <li><a href="./order-karma.html">Store</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Login</a></li>
+            </ul>
+        </nav>
+        <div class="hamburger">
+            <a href="#" onclick="toggleMenu()">
```

Comment:
Well done on adding javascript functionalities to the page. I would recommend you use event listeners rather than inline HTML events to better structure your code.

More info on adding on click events for event listeners here: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

Reply:
Thank you, I have changed it to event listener and moved the code out of HTML